apps/storybook.namekit.io/stories/Namekit/Identity.stories.tsx

```diff
@@ -35,29 +35,147 @@ const IdentityCard: React.FC<{
     <Identity.Name />
     <Identity.Address />
     <Identity.NameGuardShield />
-    <Identity.ENSProfileLink />
+    <Identity.ProfileLink>
+      <div>
+        <ENSLogo />
+        <span>View on ENS App</span>
+      </div>
+    </Identity.ProfileLink>
     <Identity.Followers />
   </Identity.Root>
 );
 
+const customAppConfig = {
+  profileLinks: {
+    getProfileURL: (address: string) => `/profiles/${address}`,
+    getProfileLink: (address: string, children: React.ReactNode) => (
+      <a href={`/profiles/${address}`}>{children}</a>
+    ),
+  },
+};
+
+const CustomAppIdentityCard: React.FC<{ address: string }> = ({ address }) => (
+  <NameKitProvider config={customAppConfig}>
+    <Identity.Root address={address}>
+      <Identity.Avatar />
+      <Identity.Name />
+      <Identity.ProfileLink>
+        <button>View Profile</button>
```

Comment:
As I mentioned before, I'm avoiding the UI side of things until we nail the DX. Swapping out components is easy, but if we need to make changes to the UI components, it could cause conflicts so I've opted not to focus on that yet.